_episodes/01-intro.md

```diff
@@ -73,8 +75,9 @@ weight_kg_text = 'weight in kilograms:'
 ~~~
 {: .language-python}
 
-## Using Variables in Python
-To display the value of a variable to the screen in Python, we can use the `print` function:
+## Using Variables and Functions in Python
+To display the value of a variable to the screen in Python, we can use the `print`
+[function]({{ page.root }}/reference.html#function):
```

Comment:
Since the current content uses `print` in showing how to use variables, do you think it makes sense to do the section about built-in functions before the using variables section?

Reply:
So, this is a difficult question and I'm not sure, tbh. We should minimize the number of times we switch context. For that, we need to consider the material we've got (variables, built-in functions, data types) and come up with a good mental model (and a story). 
Also, we can (and probably should) talk more about the data that we have and patient's data: patients' weight (floats), DOB or name (strings), age or inflammation bouts on a particular day (ints), and history of inflammation bouts (lists).

Reply:
For combining these topics (variables, built-in functions, data types), would the following presentation of material work for the "Using variables" section?:
- Variables store data; we can then use them as we would the data they represent, e.g. weight_lb = weight_kg * 2.2, etc.
- Can do more than simple arithmetic with data by using functions; these carry out specific tasks on our data. We recognise these by the parentheses that follow them. Creating our own functions is discussed in more detail later, but Python comes with a large number of built-in functions for carrying out useful tasks. For example, to display the value of a variable we can use print(value), etc.
- There are many useful built-in functions, e.g. type() tells us the type of data we have. These functions can also be combined in useful ways, e.g. print(type(value)) allows us to output the type of a variable.

This is roughly the content here, but I think explicitly explaining functions in the context of "things we can use variables for" makes it clearer here why it is being discussed and why the built-in functions are useful. A further explanation could then have its own section if required, but I think a brief explanation of (built-in) functions could fit quite naturally under "Using variables in Python" if framed as "functions can be used to achieve useful things with our data" as above. This admittedly makes the section slightly longer rather than a short addition and an extra box as currently; is there a strong reluctance to expand the Using Variables section a little to add built-in functions? I think some mention of functions would aid understanding of what we are doing in subsequent lessons (passing data to functions from numpy in the immediate next lesson, etc.)

(It is tempting if mentioning functions to also mention that variables can be used to hold their outputs using the assignment syntax already explained, e.g. using abs() to get absolute value, but this might be adding too much new content)

Reply:
> is there a strong reluctance to expand the Using Variables section a little to add built-in functions?

no, not really. I proposed to have a separate section on built-in functions so that we can devote enough time to them but the narrative can flow as if it's all one big story. I'd prefer to have a heading because when I was teaching some other SWC Python lesson (ahem gapminder), I couldn't notice that headings were helping me to quickly recall what I was going to talk about next.